Library/Homebrew/extend/pathname.rb

```diff
@@ -27,17 +27,24 @@ def abv
 
   def compute_disk_usage
     if directory?
+      scanned_files = Set.new
       @file_count = 0
       @disk_usage = 0
       find do |f|
-        if !f.directory? && !f.symlink? && f.basename.to_s != ".DS_Store"
-          @file_count += 1
-          @disk_usage += f.size
+        next if f.directory?
```

Comment:
If we want to match `gdu -s --apparent-size` this needs to be

```
-        next if f.directory?
+        if f.directory?
+          @disk_usage += f.stat.size
+          next
+        end
```
But I think it's debatable whether we want to match that or not.